stylegan_code_finder/train_with_lightning.py

```diff
@@ -125,6 +144,8 @@ def main(args: argparse.Namespace):
                         help='If the scheduler should use warm restarts')
     parser.add_argument('--wandb-project-name', default='Debug', help='The project name of the WandB project')
     parser.add_argument('--debug', action='store_true', default=False, help='Special mode for faster debugging')
+    parser.add_argument('-vi', '--validation-interval', type=float, default=0.1,
```

Comment:
I think this argument should also go into config files where the rest of the flags (that control logging) are